server/app/views/v1/grid_stats/stats.json.jbuilder

```diff
@@ -0,0 +1,15 @@
+json.from @node_stats[:from_time]
+json.to @node_stats[:to_time]
+json.stats @node_stats[:stats] do |stat|
+  json.timestamp stat[:timestamp]
+  json.data_points stat[:data_points]
+  json.cpu_usage_percent stat[:cpu_usage_percent]
+  json.memory_used_bytes stat[:memory_used_bytes]
```

Comment:
Should these be objects instead of flat keys? Example:
```json
{
  "memory": {
    "used_bytes": 1111
  }
}
```

@nevalla what do you think?